### PR TITLE
Add READMEs

### LINKED ISSUE #78: Write READMEs
Add README files to:
- the root directory
- code/part-one
- code/part-two
- code/part-two/client
- code/part-two/processor

These READMEs should include:
- guidance about how to use the repo
- instructions on how to start components
- instructions on how to run tests and complete problems
- design specs for the apps being built

### PR DESCRIPTION
**Pull request by delventhalz**
Adds READMEs to:
- `/`
- `code/part-one/`
- `code/part-two/`
- `code/part-two/client`
- `code/part-two/processor`

The READMEs should provide enough information (with docstrings and linked curriculum) for enterprising testers to attempt the sprint. Eventually video lectures and slides will be added to better supplement this information.

The PR needs to get merged ASAP. So please focus on doing a quick review for egregious typos or confusing passages. Feel free to make a note of more nitpicky stuff, but this needs to be merged by Friday afternoon or we lose Memorial Day weekend for testing.

Closes #78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,111 @@
...
+
+(づ￣ ³￣)づ
+
+
+### Solution Code
+
+In addition to the prompts and stub functions in the `master` branch, this repo
+includes a `solution` branch with completed versions of both sections. This is
+intended primarily as a way for students to check their work, and become
+familiar current best practices. If your intention is to complete the sprint,
```
Comment (therobinkim) [inline README.md]: missing word: familiar *with* current best practices

File: README.md [hunk 2]
```diff
@@ -1 +1,111 @@
...
+it is recommended that you do not use this solution code as a way to get
```
Comment (therobinkim) [inline README.md]: minor: `it is recommended` => `we recommend`

File: code/part-two/README.md [hunk 3]
```diff
@@ -1,54 +1,589 @@
...
+        - [Accept Response](#accept-response)
+        - [Cancel Offer](#cancel-offer)
+        - [Cancel Response](#cancel-response)
+- [Nightmare Mode](#nightmare-mode)
 
-## Docker Commands
+## Using Docker
+
+Docker is a virtualization tool that makes it much easier to deploy code in
+a variety of environments, with a minimum of effort. Dockerfiles create
```
Comment (therobinkim) [inline code/part-two/README.md]: remove extra word:
> with a minimum ~of~ effort

File: code/part-two/README.md [hunk 4]
```diff
@@ -1,54 +1,589 @@
...
+#### Offer
+
+```json
+{
+    "owner": <string, public key>,
+    "moji": [ <strings, moji addresses> ],
+    "responses": [
+        {
+            "approver": <string, public key>,
+            "moji": [ <strings, moji addresses> ]
```
Comment (therobinkim) [inline code/part-two/README.md]: I think lns 450, 451, 454, 455 should have double quotes around the values if we want to be consistent w/ how we represented this json information previously
```json
{
    "owner": "<string, public key>",
    "moji": [ "<strings, moji addresses>" ],
    "responses": [
        {
            "approver": "<string, public key>",
            "moji": [ "<strings, moji addresses>" ]
        }
    ]
}
```

File: code/part-two/README.md [hunk 5]
```diff
@@ -1,54 +1,589 @@
...
+is required to approve a response is stored for convenience on the response
+itself.
+
+### Addressing
+
+#### Offer
+
+An offer's address begins the same way as any other, with the cryptomoji
+namespace and a type prefix (`03` in this case). Afterwards, it uses the same
+eight character owner prefix that cryptomoji due. The final 54 characters is
```
Comment (therobinkim) [inline code/part-two/README.md]: `due` => `do`?

Comment (therobinkim) [inline code/part-two/README.md]: `characters is` => `characters are`?

File: code/part-two/README.md [hunk 6]
```diff
@@ -1,54 +1,589 @@
...
+also the first 54 characters of a SHA-512 hash. In this case, the hash generate
```
Comment (therobinkim) [inline code/part-two/README.md]: missing word: hash **is** generated

File: code/part-two/client/README.md [hunk 7]
```diff
@@ -1,22 +1,216 @@
...
+```bash
+npm run build:prod
+```
+
+You can build the dev code. Aside from more helpful error messages, this mode
+will keep running and automatically rebuild anytime you save changes:
 
 ```bash
-npm run live-server
+npm run build:prod
```
Comment (therobinkim) [inline code/part-two/client/README.md]: I think maybe this is supposed to be `build:dev`

File: code/part-two/client/README.md [hunk 8]
```diff
@@ -1,22 +1,216 @@
...
+order to run the build, there are a few commands you can use.
+
+You can build the production code:
+
+```bash
+npm run build:prod
+```
+
+You can build the dev code. Aside from more helpful error messages, this mode
+will keep running and automatically rebuild anytime you save changes:
```
Comment (therobinkim) [inline code/part-two/client/README.md]: dev mode (for webpack) is also faster at rebundling than production mode, but idk if we wanna mention that via a later commit ¯\_(ツ)_/¯

**Comment by delventhalz** [comment]
@therobinkim @hmfoster Good catches! All comments (so far) have been addressed.

**Pull request merged**